contrib/debian/dev-packages.lst

```diff
@@ -4,7 +4,7 @@ libprotobuf-dev
 libprotobuf-c-dev
 protobuf-c-compiler
 protobuf-compiler
-python-protobuf
+python3-protobuf
```

Comment:
This was the main reason I arrived here; I noticed that our Dockerfile installed `python-protubuf`, which depends on python 2, so I wondered if we could switch. Then I noticed there's other dependencies that still defaulted to python 2, so I updated those.

Haven't tested yet locally, so "abusing" CI on this repository to see if everything is ok with my change